src/libraries/System.Diagnostics.DiagnosticSource/tests/AggregationManagerTests.cs

```diff
@@ -16,6 +16,12 @@ public void GetDefaultAggregator()
 
             Assert.NotNull(val);
             Assert.Equal(val, store.GetAggregator());
+
```

Comment:
I think you could remove all the changes from this file. These unit tests are validating the behavior of the AggregatorStore type which should be indifferent to the type of Aggregator stored inside. Using LastValue was an arbitrary choice because I needed any type that derived from Aggregator.

Reply:
I reverted the changes in this file.